ax/models/torch/botorch_modular/sebo.py

```diff
@@ -233,16 +232,34 @@ def optimize(
             candidates, a tensor with the associated acquisition values, and a tensor
             with the weight for each candidate.
         """
+        # Prepare arguments for optimizer
+        optimizer_options_with_defaults = optimizer_argparse(
+            self.acqf,
+            optimizer_options=optimizer_options,
+            optimizer="optimize_acqf_homotopy",
+        )
+
+        _tensorize = partial(torch.tensor, dtype=self.dtype, device=self.device)
+        ssd = search_space_digest
+        bounds = _tensorize(ssd.bounds).t()
+
+        optimizer_options["batch_initial_conditions"] = get_batch_initial_conditions(
+            acq_function=self.acqf,
+            raw_samples=optimizer_options_with_defaults["raw_samples"],
+            inequality_constraints=inequality_constraints,
+            fixed_features=fixed_features,
+            X_pareto=self.acqf.X_baseline,
```

Comment:
We have a pyre failure. The union types in PyTorch don't play well with type checkers :(
fbcode/ax/models/torch/botorch_modular/sebo.py:251:12 Incompatible parameter type [6]: In call `get_batch_initial_conditions`, for argument `X_pareto`, expected `Tensor` but got `Union[Tensor, Module]`.

```suggestion
            X_pareto=assert_is_instance(self.acqf.X_baseline, Tensor),
```